### DISCUSSION TITLE
Can not install T-Pot with new Fedora - resolved.conf don´t exist

### DISCUSSION THREAD

**Discussion by DO2SN** [telekom-security/tpotce#1558] [Q&A] [answered]
Thanks a lot for this amazing tool. 

Scince a few days I´m trying to install t-pot. 

I choosed Fedora workstation 40 as it is listed as a supported distro. I installed Fedora with a standalone PC, installed latest updates, set up SSH and RDP. 

Tried out some things... Internet is available. DNS is resolved correctly. 

I worked along readme file and tried to install tpotce. 

Installer interrupts with failure modify sublistener in resolved.conf. 

I went after the error and found out that resolved.conf file in /systemd/conf is not existing after all. 

![Fehler_Resolved_conf](https://github.com/telekom-security/tpotce/assets/171075854/ee7f20ca-5890-4892-8977-ffec938356ab)

I´m not very used to llinux... that´s why I tried out Fedora with standalone install. I want to try out tpotce to get more comfortable with things. 

Thank you so much for commuity support in advance!

Regards, Steve

**Comment by t3chn0m4g3** [comment]
Did you install this ISO (Fedora-Server-netinst-x86_64-39-1.5.iso) as linked in the T-Pot ReadMe or did you use the Workstation one?

**Reply by DO2SN** [reply]
Dear t3ch0m4g3, 
indeed I have installed Fedora 40 Workstation Edition by mistake.  Attached you will find my system config. 

So will there be an installer anyday in the future or do I have to use the linked one  Fedora.iso?

![Fedora_Version](https://github.com/telekom-security/tpotce/assets/171075854/406c39e4-bd4a-4138-87bb-f848af85b9d4)

Appreciate your great support and all the best to the community. 

Regards, Steve

**Reply by t3chn0m4g3** [reply, marked as answer]
Please follow the ReadMe to the linked supported images and do not install any extra packages other than SSH.
